scooping_cv/README.md

```diff
@@ -1,2 +1,69 @@
 # scooping_cv
 SS19 Computer Vision - See the Icecream
+
+## Tracking dependencies
+
+**Bayesian Object Tracking**     
+follow the guide here: https://github.com/bayesian-object-tracking/getting_started
+
+The .obj files are located in scooping_cv/meshes/tracking
+
+To get real-time results, a GPU should be used.
+
+
+**Eigen 3.2**      
+In order to install Eigen 3.2.10, download http://bitbucket.org/eigen/eigen/get/3.2.10.tar.bz2.
+
+run following codes:
+
+$ cd catkin_ws/src
+
+download using
+
+$ wget http://bitbucket.org/eigen/eigen/get/3.2.10.tar.bz2
+
+$ tar -xf 3.2.10.tar.bz2    
+$ cd eigen-eigen-b9cd8366d4e8    
+$ mkdir build   
+$ cd build    
+$ cmake ..   
+$ sudo make install
+
+## Camera choice
+
+THe problem with the ZED camera is that it cannot track homogeneous surfaces, e.g. the cup. The cup itself will be detected as flat surface, which corrupts the bayesian object tracking.
+
+Using the Intel Realsense R200 (which has an infrared depth sensor), the cup is recognized without problems. However, the scooping tool has a special surface that does not go well with infrared - it is not detectable in the depth cloud. This needs to be solved. An alternative would be using AR Tags.
```

Comment:
Didn't wrapping it with paper work?

Reply:
Only wrapping the handle is not the best solution, the tracking is still a bit inaccurate.. But true that part of the readme needs a little update :)